piqueserver/server.py

```diff
@@ -282,11 +271,11 @@ def __init__(self, interface, config):
         if not os.path.isabs(logfile):
             logfile = os.path.join(cfg.config_dir, logfile)
         if logfile.strip():  # catches empty filename
+            os.makedirs(os.path.dirname(logfile), exist_ok=True)
```

Comment:
Hey, I had no idea that kwarg exists! Unfortunately, it's py3 only.

Reply:
oh that's sad... might still need that helper function then.